crate_universe/src/rendering/calls.rs

```diff
@@ -0,0 +1,90 @@
+use std::collections::BTreeSet as Set;
+
+use serde::ser::{SerializeStruct, SerializeTupleStruct, Serializer};
+use serde::Serialize;
+use serde_starlark::{Error as StarlarkError, FunctionCall, MULTILINE, ONELINE};
+
+#[derive(Serialize)]
+#[serde(untagged)]
+pub enum Starlark {
+    Package(Package),
+    ExportsFiles(ExportsFiles),
+    Filegroup(Filegroup),
+    Alias(Alias),
+
+    #[serde(skip_serializing)]
+    Comment(String),
+}
+
+pub struct Package {
+    pub default_visibility: Set<String>,
+}
+
+pub struct ExportsFiles {
+    pub paths: Set<String>,
+    pub globs: Glob,
+}
+
+#[derive(Serialize)]
+#[serde(rename = "filegroup")]
+pub struct Filegroup {
+    pub name: String,
+    pub srcs: Glob,
+}
+
+#[derive(Serialize)]
+#[serde(rename = "glob")]
+pub struct Glob(pub Set<String>);
```

Comment:
All the content here seems to be simpler versions of things in [crate::utils::starlark](https://github.com/bazelbuild/rules_rust/blob/0.15.0/crate_universe/src/utils/starlark.rs). Any reason not to move it to the same module?

Reply:
Done.